DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -14,8 +14,9 @@
     @if (errorHasOccurred) {
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
     }
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove delegate enrolment from @Model.CourseName</h1>
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove enrolment</h1>
     <h2 class="nhsuk-heading-l word-break">Delegate: @Model.Name</h2>
+    <h2 class="nhsuk-heading-l word-break">Activity: @Model.CourseName</h2>

```

Comment:
Comparing the screenshot to other remove confirmation pages on the site, it looks like there's usually a little more margin under the title, which would be good to have here for consistency. I think it often comes from putting the h1 in it's own row - see RemoveGroupCourse.cshtml and RemoveGroupDelegate.cshtml for examples